hspec-core/src/Test/Hspec/FailureReport.hs

```diff
@@ -18,6 +22,9 @@ data FailureReport = FailureReport {
 } deriving (Eq, Show, Read)
 
 writeFailureReport :: FailureReport -> IO ()
+#ifdef __GHCJS__
+writeFailureReport _ = return ()
```

Comment:
Can we have a comment here, describing the reason for this + e.g. url to corresponding issues.